workers/unity/Packages/com.improbable.gdk.tools/LocalLaunch.cs

```diff
@@ -204,7 +204,7 @@ public static void LaunchLocalDeployment()
             BuildConfig();
 
             var command = Common.SpatialBinary;
-            var commandArgs = "local launch";
+            var commandArgs = "local launch --enable_pre_run_check=false";
```

Comment:
Is this definitely needed?

Reply:
yes because it will otherwise call `prepare-for-run` under the hood - which will not work if schema is not in the default place (e.g. within the packages)